### DISCUSSION TITLE
Error when pulling Beta 10

### DISCUSSION THREAD

**Discussion by gregeeh** [blakeblackshear/frigate#5831] [Ask A Question] [answered]
Hi all,

I'm getting the following error when trying to pull Beta 10.

```
root@omv:~# docker pull ghcr.io/blakeblackshear/frigate:0.12.0-beta10
0.12.0-beta10: Pulling from blakeblackshear/frigate
3f9582a2cbe7: Already exists
c826d1cb7406: Retrying in 1 second
4cfc8a6965c0: Retrying in 1 second
43e2bddea720: Retrying in 1 second
6c76ffc8f6a5: Waiting
057a40b79daf: Waiting
0bd0c88fea2e: Waiting
error pulling image configuration: download failed after attempts=6: x509: certificate is valid for *.githubassets.com, githubassets.com, not pkg-containers.githubusercontent.com
```
Any ideas appreciated.

TIA

**Comment by NickM-27** [comment, marked as answer]
Looks like a time issue or something getting certificates?

**Reply by gregeeh** [reply]
Seems to be working now.  Strange.

**Comment by morganchristiansson** [comment]
I got here via github search and I'm receiving same error for pulling an unrelated project.

Visiting https://pkg-containers.githubusercontent.com/ in browser also gives cert error. Must be github issue not sure it's reported correctly.
